search/tests/utils.py

```diff
@@ -48,34 +50,32 @@ class ForceRefreshElasticSearchEngine(ElasticSearchEngine):
     so that tests can relaibly search right afterward
     """
 
-    def index(self, doc_type, sources, **kwargs):
-        kwargs.update({
-            "refresh": True
-        })
-        super(ForceRefreshElasticSearchEngine, self).index(doc_type, sources, **kwargs)
+    def index(self, sources, **kwargs):
+        kwargs["refresh"] = True
+        super(ForceRefreshElasticSearchEngine, self).index(sources, **kwargs)
 
-    def remove(self, doc_type, doc_ids, **kwargs):
-        kwargs.update({
-            "refresh": True
-        })
-        super(ForceRefreshElasticSearchEngine, self).remove(doc_type, doc_ids, **kwargs)
+    def remove(self, doc_ids, **kwargs):
+        kwargs["refresh"] = True
+        super(ForceRefreshElasticSearchEngine, self).remove(doc_ids, **kwargs)
 
 
 class ErroringSearchEngine(MockSearchEngine):
     """ Override to generate search engine error to test """
 
-    def search(self,
-               query_string=None,
-               field_dictionary=None,
-               filter_dictionary=None,
-               **kwargs):  # pylint: disable=arguments-differ
+    def search(
+            self,
```

Comment:
ditto

Reply:
if I turn method signature parameters to the left code checkers are failed
`pylint and pep8`